Makefile

```diff
@@ -5,9 +5,9 @@
 all: modl
 
 modl:
-	cd $(PWD)/mod/ && nrnivmodl
+	cd hnn_core/mod/ && nrnivmodl
```

Comment:
the [latest instructions](https://github.com/jonescompneurolab/hnn-core/blob/master/CONTRIBUTING.rst) say you can do:

```sh
$ python setup.py build_mod
```

did that not work for you?

Reply:
I just didn't follow the instructions :(